src/organizer/main.py

```
@@ -0,0 +1,42 @@
+) -> None:
+    if show_logs:
+        setup_logging()
+        logger.info("Starting organizer under path: %s", path)
+    try:
+        llm = IntelligentFileOrganizer(llm_model)
+        organizer = Organizer(path, llm_client=llm)
+        organizer.organize()
+    except Exception as e:
+        logger.error("Error organizing files: %s", e)
+        typer.secho(f"Error organizing files: {e}", fg=typer.colors.RED, err=True)
+        raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)